### ISSUE TITLE
Does batch size change while traininig with elastic agent?

### ISSUE THREAD

**Issue by Esthesia** [SeanNaren/deepspeech.pytorch#686]
Thanks for sharing code with torch elastic training.
I have one question about node preemption with batch size.
If the gloabal batch size is 64 with 4 nodes (per 1 GPU), then the local batch size will be 16.
When one node is preempted, how the global batch size will be changed and the effect of changing (e.g., loss) is significant?
I tried to find out the detailed information with torch elastic but cannot find it, so I asked here because deepspeech already took a real experiment.
Thank you for reading this.
